1.9/administration/logging/aggregating/logging-api.md

```diff
@@ -0,0 +1,132 @@
+
+# REST API
+#### Endpoints:
+- `/range/` get a range of logs matching the request query.
+- `/range/framework/{framework_id}/executor/{executor_id}/container/{container_id}` get a range of application logs.
+- `/stream/` tail logs keeping the connection opened, implements Server Sent Events.
+- `/stream/framework/{framework_id}/executor/{executor_id}/container/{container_id}` tail application logs.
+- `/fields/<field>` returns all possible unique values for a specific `<field>`.
+
+#### Request Header Accept
+- `text/plain`, `text/html`, `*/*` request logs in text format, ending with `\n`.
+- `application/json` request logs in JSON format.
+- `text/event-stream` request logs in Server-Sent-Events format.
+
+#### Request Header Last-Event-ID
+If `Last-Event-ID` is set dcos-log will use it as a cursor position. `Last-Event-ID` header works with `/stream/` endpoints only.
+
+NOTE: Accept header `text/event-stream` cannot be used with `/fields/<field>` endpoint.
+
+#### Response Header X-Journal-Skip-Prev
+`X-Journal-Skip-Prev` indicates how many entries we actually skipped backwards from the current position.
+
+Example:
+```
+request: `?cursor=<cursor string>&skip_prev=1000`
+response includes the headers: `X-Journal-Skip-Prev: 100`, `X-Journal-Skip-Next: 0`
+
+This means that there are only 100 entries before the given cursor position and we reached the top of the journal.
+
+```
+
+#### Response Header X-Journal-Skip-Next
+
+`Header X-Journal-Skip-Next` indicates how many entries we actually skipped forward from the current position.
+
+see `X-Journal-Skip-Prev` example above.
+
+#### GET parameters:
+- `?filter=FIELD:value` add match.
+- `?limit=N` limit number of entries.
+- `?skip_next=N` skip forward number of entries from the current cursor position.
+- `?skip_prev=N` skip backwards number of entries from the current cursor position.
+- `?cursor=CURSOR` set cursor position. (Special characters must be escaped).
+- `?read_reverse=true` read the journal in opposite direction (bottom to top).
+
+where
+- `FIELD`, `value` and `CURSOR` are strings.
+- `N` is uint64.
+
+NOTE:
+- It is possbile to move to the tail of the journal. If the `?cursor` parameter is not used then we consider the cursor
+  is pointing to a head of the journal. `?skip_prev=1` can be used to move to the tail of the journal (very last entry). If you need to read last 10 entries you should use `?skip_prev=10`.
+- Parameter `?limit` cannot be used with `/stream/` endpoint.
+
+#### Response codes:
+- `200` OK.
+- `204` Content not found, returned if no entries matching requesting filters.
+- `400` Bad request, returned if request is incorrect.
+- `500` Internal server error.
+
+# CLI flags
+```
+Usage of dcos-log:
+  -config string
+       	Use config file.
+  -config-json-schema string
+       	Use a custom json schema.
+  -port int
+       	Set TCP port. (default 8080)
+  -verbose
+       	Print out verbose output.
+```
+
+# Examples:
```

Comment:
i think we should also exclude the examples, or adjust the endpoints with a prefix from my first comment